projects/kit/components/slider/slider.component.ts

```diff
@@ -84,6 +95,31 @@ export class TuiSliderComponent {
         this.el.value = `${newValue}`;
     }
 
+    @Input({
+        alias: 'min',
+        transform: (x: number | string) => coerceNumberProperty(x), // TODO: remove `transform` in v5
+    })
+    protected set minSetter(x: number) {
+        isWritableSignal(this.min) && this.min.set(x);
+    }
+
+    @Input({
+        alias: 'max',
+        transform: (x: number | string) => coerceNumberProperty(x), // TODO: remove `transform` in v5
+    })
+    protected set maxSetter(x: number) {
+        isWritableSignal(this.max) && this.max.set(x);
```

Comment:
Can it be `this.max.set?.(x)`?

Reply:
It will throw ts-error:

```
Error: projects/kit/components/slider/slider.component.ts:111:18 - error TS2339:
Property 'set' does not exist on type 'WritableSignal<number> | Signal<number>'.
  Property 'set' does not exist on type 'Signal<number>'.

111         this.max.set?.(x);
                     ~~~

```

Reply:
Ok. Would be nice to get rid of duality of Writable and non-Writable then.